serverless/event_workflows/serverless-subs.adoc

```diff
@@ -0,0 +1,30 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-subs"]
+= Using subscriptions
+include::modules/common-attributes.adoc[]
+:context: serverless-subs
+
+toc::[]
+
+After events have been sent to a channel from an event source or producer, these events can be sent to multiple Knative services, or other sinks, by using a subscription.
+
+image::serverless-event-channel-workflow.png[Channel workflow overview]
+
+If a subscriber rejects an event, there are no re-delivery attempts by default. Developers can configure xref:../../serverless/event_workflows/serverless-event-delivery.adoc#serverless-event-delivery[re-delivery attempts] by modifying the `delivery` spec in a `Subscription` object.
+
+[id="serverless-subs-creating-subs"]
+== Creating subscriptions
+
+Developers can create subscriptions that allow event sinks to subscribe to channels and receive events directly.
```

Comment:
What does it mean to 'receive events directly' ? What other possibilities do I have to receive events ? (I know this is probably via triggers, but  I wouldn't call that "indirectly").

Reply:
I'll remove the word directly here